pymtl3/passes/PassGroups.py

```diff
@@ -43,8 +45,11 @@
   DynamicSchedulePass(),
   CLLineTracePass(),
   VcdGenerationPass(),
+  #WaveGenPass(),
+  CollectSignalPass(),
+  PrintWavePass(),
   SimpleTickPass(),
-  Component.lock_in_simulation
+Component.lock_in_simulation
```

Comment:
Can you take a pass to make sure lines like this are properly aligned? We would like to make sure the codebase has a consistent style by checking each PR in details...